pkg/api/types.go

```diff
@@ -32,7 +34,15 @@ type Broker struct {
 	Status BrokerStatus
 }
 
+const (
+	// DescriptionKey is the key of an annotation that holds the brief
+	// description of an API resource
+	DescriptionKey = "alpha.service-catalog.kubernetes.io/description"
```

Comment:
@pmorie Feel free to use `description` unprefixed. That's a general convention we should promote.